en/ros/mavros_offboard.md

```diff
@@ -2,19 +2,19 @@
 
 > **Caution** Offboard control is dangerous. If you are operating on a real vehicle be sure to have a way of gaining back manual control in case something goes wrong.
 
-The following tutorial will run through the basics of offboard control through mavros as applied to an Iris quadcopter simulated in Gazebo. At the end of the tutorial, you should see the same behaviour as in the video below, i.e. a slow takeoff to an altitude of 2 meters.
+The following tutorial will run through the basics of `OFFBOARD` control through MAVROS as applied to an Iris quadcopter simulated in Gazebo with SITL running. At the end of the tutorial, you should see the same behaviour as in the video below, i.e. a slow takeoff to an altitude of 2 meters.
```

Comment:
In fairness, as stated below (https://github.com/PX4/Devguide/pull/209#issuecomment-312496575) some docs capitalise [Flight Modes (Overview)](https://docs.px4.io/en/config/flight_mode.html) and others don't [Basic Configuration > Flight Modes](https://docs.px4.io/en/config/flight_mode.html).

I have a leaning towards capitalisation because (hence [RTL](https://docs.px4.io/en/flight_modes/rtl.html), [LAND](https://docs.px4.io/en/flight_modes/land.html)) but it doesn't really matter as long as we are consistent.

@mhkabir Thoughts? (if you have a strong preference let me know. I can push whatever we agree into style guide/out through docs)